pom.xml

```diff
@@ -85,6 +141,31 @@
                     <compatibleSinceVersion>2.0</compatibleSinceVersion>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>copy-binary-files</id>
+                        <phase>process-test-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.codehaus.groovy</groupId>
+                                    <artifactId>groovy-binary</artifactId>
+                                    <version>2.4.13</version>
+                                    <type>zip</type>
+                                    <outputDirectory>${project.build.testOutputDirectory}</outputDirectory>
```

Comment:
Perhaps worth stressing this is for testing only.

Reply:
Besides use of `testOutputDirectory`?

Reply:
Well, the 20+ lines requires somewhat deep inspection to uncover the true purpose which can be summarized in one sentence.

Reply:
Agreed, I added a comment about it above.